Neo4j.Driver/Neo4j.Driver/Preview/Mapping/MappingFailedException.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Copyright (c) "Neo4j"
+// Neo4j Sweden AB [https://neo4j.com]
+// 
+// Licensed under the Apache License, Version 2.0 (the "License").
+// You may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// 
+//     http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System;
+
+namespace Neo4j.Driver.Preview.Mapping;
+
+public class MappingFailedException : Exception
```

Comment:
Should this inherit from Neo4jException?

Reply:
I think it would make sense that it does. I don't think there are any other exceptions that don't.

Reply:
Done.